### PR TITLE
Hydrostatic-aware vapor pressure; >7km simple well cost correlation; other fixes (v3.4.4)

### PR DESCRIPTION
**Pull request by softwareengineerprogrammer**
1. https://github.com/NREL/GEOPHIRES-X/issues/118 - see discussion in https://github.com/softwareengineerprogrammer/GEOPHIRES-X/pull/12
1. [Implement missing simple well cost correlation for non-AGS wells >7km depth](https://github.com/NREL/GEOPHIRES-X/pull/126/files#r1493823487) ([commit](https://github.com/NREL/GEOPHIRES-X/pull/126/commits/a463d1a7f9943a5e59fe49507bcd2796264f4a77))
1. https://github.com/NREL/GEOPHIRES-X/issues/65
1. Address https://github.com/NREL/GEOPHIRES-X/issues/120

### CODE REVIEW

File: src/geophires_x/Economics.py [hunk 1]
```diff
@@ -1766,40 +1766,44 @@ def Calculate(self, model: Model) -> None:
...
                                                        self.Nonvertical_drilling_cost_per_m.value * model.wellbores.Nonvertical_length.value)) * 1E-6
                             else:
                                 self.C1well = (2 * self.Vertical_drilling_cost_per_m.value * (
-                                        model.reserv.depth.value * 1000.0)) * 1E-6
-                    else:  # Coaxial
-                        self.C1well = ((self.Vertical_drilling_cost_per_m.value * (model.reserv.depth.value * 1000.0)) +
+                                        model.reserv.depth.value.quantity().to('m').magnitude)) * 1E-6
+                    else:
+                        # Coaxial
+                        self.C1well = ((self.Vertical_drilling_cost_per_m.value * (model.reserv.depth.quantity().to('m').magnitude)) +
```
Comment (softwareengineerprogrammer) [inline src/geophires_x/Economics.py:1800]: [Implement missing simple well cost correlation for non-AGS wells >7km depth](https://github.com/NREL/GEOPHIRES-X/pull/126/commits/a463d1a7f9943a5e59fe49507bcd2796264f4a77)

File: src/geophires_x/Economics.py [hunk 2]
```diff
@@ -1766,40 +1766,44 @@ def Calculate(self, model: Model) -> None:
...
                                        (
                                                self.Nonvertical_drilling_cost_per_m.value * model.wellbores.Nonvertical_length.value)) * 1E-6
 
+                else:
+                    self.C1well = self.Vertical_drilling_cost_per_m.value * model.reserv.depth.quantity().to(
+                        'm').magnitude * 1E-6
```
Comment (softwareengineerprogrammer) [inline src/geophires_x/Economics.py:1806]: [Implement missing simple well cost correlation for non-AGS wells >7km depth](https://github.com/NREL/GEOPHIRES-X/pull/126/commits/a463d1a7f9943a5e59fe49507bcd2796264f4a77)

File: src/geophires_x/Outputs.py [hunk 3]
```diff
@@ -371,7 +371,11 @@ def PrintOutputs(self, model: Model):
                     f.write(f"      Initial Net Electricity Generation:               {model.surfaceplant.NetElectricityProduced.value[0]:10.2f} " + model.surfaceplant.NetElectricityProduced.PreferredUnits.value + NL)
                     f.write(f"      Average Annual Total Electricity Generation:      {np.average(model.surfaceplant.TotalkWhProduced.value/1E6):10.2f} GWh" + NL)
                     f.write(f"      Average Annual Net Electricity Generation:        {np.average(model.surfaceplant.NetkWhProduced.value/1E6):10.2f} GWh" + NL)
-                    if model.wellbores.PumpingPower.value[0] > 0.0: f.write(f"      Initial pumping power/net installed power:        {(model.wellbores.PumpingPower.value[0]/model.wellbores.PumpingPower.value[0]*100):10.2f} %" + NL)
+
+                    if model.wellbores.PumpingPower.value[0] > 0.0:
+                        ipp_nip = model.wellbores.PumpingPower.value[0] / model.surfaceplant.NetElectricityProduced.value[0]
+                        f.write(f'      Initial pumping power/net installed power:        {(ipp_nip*100):10.2f} %\n')
```
Comment (softwareengineerprogrammer) [inline src/geophires_x/Outputs.py:377]: https://github.com/NREL/GEOPHIRES-X/issues/65

File: src/geophires_x/Parameter.py [hunk 4]
```diff
@@ -212,7 +212,11 @@ class listParameter(Parameter):
             which means that any value is valid by default
     """
 
-    value: List[float] = field(default_factory=list) # FIXME set from DefaultValue
+    def __post_init__(self):
+        if self.value is None:
+            self.value:str = self.DefaultValue
```
Comment (softwareengineerprogrammer) [inline src/geophires_x/Parameter.py:217]: https://github.com/NREL/GEOPHIRES-X/issues/120

File: src/geophires_x/WellBores.py [hunk 5]
```diff
@@ -496,7 +496,7 @@ def InjPressureDropAndPumpingPowerUsingIndexes(
         # Minimum production pump inlet pressure and minimum wellhead pressure
         Pminimum_kPa = vapor_pressure_water_kPa(
             Trock_degC,
-            # TODO pass pressure https://github.com/NREL/GEOPHIRES-X/issues/118
+            pressure=quantity(Phydrostaticcalc_kPa, 'kPa'),
```
Comment (softwareengineerprogrammer) [inline src/geophires_x/WellBores.py:499]: https://github.com/NREL/GEOPHIRES-X/issues/118

File: tests/test_geophires_x.py [hunk 6]
```diff
@@ -217,6 +219,43 @@ def _get_unequal_dicts_approximate_percent_difference(self, d1: dict, d2: dict)
 
         return None
 
+    def test_clgs_depth_greater_than_5km(self):
```
Comment (softwareengineerprogrammer) [inline tests/test_geophires_x.py:222]: Relevant to https://github.com/NREL/GEOPHIRES-X/issues/125

**Review by malcolm-dsider — APPROVED** [review]

**Pull request merged**